_1327/user_management/tests.py

```diff
@@ -134,7 +134,7 @@ def test_impersonate_any_user(self):
 			response = response.follow()
 			self.assertEqual(response.status_code, 200)
 
-			self.assertIn("Logged in as {username}".format(username=user.username), response.body.decode('utf-8'))
+			self.assertIn("{username}".format(username=user.username), response.body.decode('utf-8'))
```

Comment:
I guess we should change this `username=user.username` to  this `username=user.get_full_name()`